__tests__/impersonation.spec.tsx

```diff
@@ -116,4 +117,18 @@ describe('Impersonation', () => {
     const root = container.querySelector('[data-workos-impersonation-root]');
     expect(root).toHaveStyle({ backgroundColor: 'red' });
   });
+
+  it('should should sign out when the Stop button is called', async () => {
+    (useAuth as jest.Mock).mockReturnValue({
+      impersonator: { email: 'admin@example.com' },
+      user: { id: '123', email: 'user@example.com' },
+      organizationId: null,
+      loading: false,
+    });
+
+    render(<Impersonation />);
+    const stopButton = await screen.findByText('Stop');
```

Comment:
nit: i was confused why we were clicking 'Stop' instead of a "log out"/signout button. (is there a non-impersonation spec that we could [additionally?] add this to?

Reply:
Is this here what you meean?

https://github.com/workos/authkit-nextjs/pull/183/files#diff-a8c3e17afcc9d081a13c30f8987cc7d7c70b8fb58c784e09238edb9344224749R398

Reply:
oops yeah that's what i'm talking about